service/src/main/java/uk/nhs/adaptors/oneoneone/cda/report/mapper/CompositionMapper.java

```diff
@@ -110,11 +113,33 @@ public Composition mapComposition(POCDMT000002UK01ClinicalDocument1 clinicalDocu
             }
         }
 
+        if (!carePlans.isEmpty()) {
```

Comment:
You don't have to check for that. Loop won't be executed if list is empty

Reply:
done